### PR TITLE
LUN-252-2: Fix failing build android step

### PR DESCRIPTION
**Pull request by SteffiStuffel**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

File: android/settings.gradle [hunk 1]
```diff
@@ -2,17 +2,4 @@ rootProject.name = 'lunes-app-android'
 
 apply from: file("../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesSettingsGradle(settings)
 
-include ':@react-native-community_async-storage'
```
Comment (klinzo) [inline android/settings.gradle:5]: Why did you remove those?

File: tsconfig.json [hunk 2]
```diff
@@ -17,7 +17,5 @@
     "types": ["jest", "node", "react", "styled-components-react-native"],
     "skipLibCheck": true
   },
-  "exclude": [
-    "node_modules", "babel.config.js", "metro.config.js", "jest.config.js", "tools"
-  ]
+  "include": ["src/**/*", "./jest.setup.ts", "index.js", "jest.config.ts", "metro.config.ci.js", "metro.config.js"]
```
Comment (klinzo) [inline tsconfig.json:20]: This e.g. excludes `App.tsx`, right? Why not exclude like we did before?

**Review by klinzo — APPROVED** [review]
Not tested

File: android/settings.gradle (hunk 1 above)
Comment (SteffiStuffel) [reply android/settings.gradle:5]: Because it is not needed.

File: tsconfig.json (hunk 2 above)
Comment (SteffiStuffel) [reply tsconfig.json:20]: No, `**` matches zero or more directories. Changed it so it is more similar to integreat.

**Review by f1sh1918 — APPROVED** [review]
